src/file/file.ts

```diff
@@ -85,13 +85,13 @@ function findNextFileNumber(folderPath: string, vault: Vault) {
   return nextNumber === 0 ? "" : nextNumber.toString();
 }
 
-function getNewFileContent(template: Template): string {
+function getNewFileContent(template: Template, hashtag: string): string {
   switch (template) {
     case Template.CUSTOM:
-      return customFormat;
+      return hashtag + '\n' + customFormat;
     case Template.FRONTMATTER:
-      return frontmatterFormat;
+      return hashtag + '\n' + frontmatterFormat;
```

Comment:
It will break frontmatter, as it should be placed in the very top of the file (read more in https://notes.nicolevanderhoeven.com/obsidian-playbook/Using+Obsidian/03+Linking+and+organizing/YAML+Frontmatter). 

Generally I think it is better to keep hashtags in the end of the file

Reply:
thank you, fixed it